app/src/main/java/org/room76/apollo/rooms/RoomsFragment.java

```diff
@@ -228,9 +266,25 @@ public void onClick(View v) {
                 int position = getAdapterPosition();
                 Room room = getItem(position);
                 mItemListener.onRoomClick(room);
+            }
+        }
+
+        public class WithoutImageViewHolder extends ViewHolder {
```

Comment:
We can simplify and use ViewHolder as WithoutImageViewHolder

Reply:
Yep. For now, while differences between WithoutImageViewHolder and FullViewHolder are only in one side.
And logically it will not be correct, too.